demos/nodejs/index.js

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable no-console */
 
+'use strict'
+
 const fs = require('fs')
-const tus = require('../..')
+const tus = require('../..') // eslint-disable-line import/no-unresolved
```

Comment:
What is this eslint comment about? The files should be present once the build scripts has been run?

Reply:
Ah that might be because I didn't build before running the linter indeed.